fog/ledger/server/src/metrics.rs

```diff
@@ -32,3 +43,74 @@ lazy_static! {
     )
     .expect("metric cannot be created");
 }
+
+pub struct MetricsUpdateThread {
```

Comment:
I'm up for suggestions here, the fog ledger router doesn't have a consistent place that it's checking the block height, so a consistent polling seemed like the best approach.

Reply:
Moved logic into the main thread loop

Reply:
Our current minimum resolution on monitoring is 30s. If we're polling, we only need to update on that frequency.

Reply:
These updates are cheap, they only update an internal counter (that will get polled at the 30s interval)